src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -229,6 +229,13 @@ def update_site_configs(resource_group_name, name, slot=None,
                         use32_bit_worker_process=None,  # pylint: disable=unused-argument
                         app_command_line=None):  # pylint: disable=unused-argument
     configs = get_site_configs(resource_group_name, name, slot)
+    if linux_fx_version:
+        linux_fx_version_lower = linux_fx_version.strip().lower()
+        if linux_fx_version_lower.startswith('docker|'):
```

Comment:
if this variable is used only once here, you can combine line 233 and 234 to save one assignment.
```
if linux_fx_version.strip().lower().startswith('docker|'):
```

Reply:
Sounds good